### PR TITLE
Document on building DIGITS on Windows

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
A brief readme style document on how to setup DIGITS w/ its dependencies to have a working DIGITS server on Windows.

### CODE REVIEW

**Comment by lukeyeager** [comment]
Looks pretty good, thanks for the doc!

* Let's use less `<hr>`'s (`-----`)
* It's nice to put each sentence on a new line for the sake of diffs. Blank lines between lines of text mark paragraphs.
* That last bit of Python - should we pull that change into DIGITS? If not, add Python syntax highlighting with "```py"

**Comment by lukeyeager** [comment]
Now that I've merged #858, you can go add a link to your document from `BuildDigits.md`.

**Comment by IsaacYangSLA** [comment]
@lukeyeager , thanks.  Let me do a little massage on my document and update the PR.  I will also put a link inside your BuildDigits.md file.

File: docs/BuildDigits.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ If those don't work for you for some reason, these instructions will walk you th
 On other platforms, installation of DIGITS itself should be pretty simple since it is a pure Python project.
 The difficulty will come from installing and configuring Caffe and/or Torch7.
 Though other platforms are not supported officially, users have successfully installed DIGITS and Caffe on CentOS, OSX and even Windows (using the `windows` branch on BVLC/caffe).
+The [document](BuildDigitsWindows.md) provides instructions on how to set up DIGITS and its dependencies on Windows.
```
Comment (IsaacYangSLA) [inline docs/BuildDigits.md]: Thanks.  I just applied the diff and commit the results.

**Pull request merged**